core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -2132,6 +2174,7 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
     synchronized (mRootLock) {
       Inode inode = getInode(srcPath);
       if (inode == null) {
+        mMasterSource.incRenameOps();
```

Comment:
Thanks for reviewing this, @apc999 . As Calvin and I discussed, we agreed to use metrics like *Ops to record how many times that operation was attempted. So the counter is incremented no matter if it is successful or not. Most of the time, we simply increment the *Ops counter in the beginning of the method like in MasterInfo.createFileInternal method. Since rename() may throw an exception before calling renameInternal, I also increment the counter here. Let me add some comments to MasterSource to clarify the *Ops counters.